include/MobileCoreServices/MobileCoreServices.h

```diff
@@ -19,3 +19,4 @@
 #import <StubIncludes.h>
 #import <MobileCoreServices/MobileCoreServicesExport.h>
 #import <MobileCoreServices/UTType.h>
```

Comment:
Yes, i think it is needed for the module map.

---
In reply to: [94506490](https://github.com/Microsoft/WinObjC/pull/1622#discussion_r94506490) [](ancestors = 94506490)